src/elements/public/Admin/Log/ErrorEntry.ts

```diff
@@ -0,0 +1,444 @@
+import { ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { NucleonElement } from '../../NucleonElement';
+import * as FoxySDK from '@foxy.io/sdk';
+import { css, CSSResult, CSSResultArray, LitElement, PropertyDeclarations, TemplateResult } from 'lit-element';
+import { html } from 'lit-html';
+import { Themeable } from '../../../../mixins/themeable';
+import { CheckboxElement } from '@vaadin/vaadin-checkbox';
+import { ButtonElement } from '@vaadin/vaadin-button';
+
+type Rel = FoxySDK.Backend.Rels.ErrorEntry;
+type Data = FoxySDK.Core.Resource<Rel, undefined>;
+
+export class ErrorEntry extends ScopedElementsMixin(NucleonElement)<Data> {
+
+  static get styles(): CSSResult | CSSResultArray {
+    return [
+      Themeable.styles,
+      css`
+        .nohide.fadeout {
+          animation-name: initial;
+        }
+        .fadeout {
+          animation-duration: 0.2s;
+          animation-name: out;
+          animation-fill-mode: forwards;
+          animation-timing-function: ease-in;
+        }
+        @keyframes out {
+          0% {
+            transform: translateX(0);
+            max-height: 120px;
+            opacity: 1;
+          }
+          99% {
+            transform: translateX(-55px);
+            max-height: 40px;
+            opacity: 0;
+            position: relative;
+          }
+          to {
+            left: -100vw;
+            position: absolute;
+          }
+        }
+      `
+    ];
+  }
+
+  static get properties(): PropertyDeclarations {
+    return {
+      ...super.properties,
+      open: { type: Boolean },
+      nohide: { type: Boolean },
+    };
+  }
+
+  public static get scopedElements(): ScopedElementsMap {
+    return {
+      'iron-icon': customElements.get('iron-icon'),
+      'foxy-i18n': customElements.get('foxy-i18n'),
+      'vaadin-button': ButtonElement,
+      'vaadin-checkbox': CheckboxElement,
+      'foxy-custom-box': Box,
+      'foxy-customer-info': CustomerInfo,
+      'foxy-client-info': ClientInfo,
+      'foxy-transaction-info': TransactionInfo,
+      'foxy-params-viewer': URLSearchParamsViewer,
+    };
+  }
+
+  open: boolean;
+
+  nohide = false;
+
+  constructor() {
+    super();
+    this.open = false;
+  }
+
+  render(): TemplateResult {
+    if (!this.data) {
+      return html`
+        <div class="absolute inset-0 flex items-center justify-center">
+          <foxy-spinner
+              data-testid="spinner"
+              class="p-m bg-base shadow-xs rounded-t-l rounded-b-l"
+              layout="horizontal"
+              >
+          </foxy-spinner>
+        </div>
+      `;
+    } else {
+      return html`
+        <div aria-busy=${this.in('busy')}
```

Comment:
**suggestion:** use details/summary

Looking at the storybook, I realise that I’ve misinterpreted your question about the checkbox (sorry!), but the point about using details/summary still stands. I think it would also be more accessible with better screen reader announcements and a larger click area (users need to be able to expand the card by clicking the header).

Reply:
08e929aead91441cee603210277360129db7d002